manifests/server/late_initdb.pp

```diff
@@ -0,0 +1,37 @@
+# @api private
```

Comment:
please also add a `@summary`  here

Reply:
Done (other private classes do not have one but it should not hurt to tell what the class is about) :smile: